templates/miq-template-ext-db.yaml

```diff
@@ -107,7 +107,7 @@ objects:
       spec:
         containers:
         - name: manageiq
```

Comment:
I don't think so.  I don't want to give it a name that suggests it is limited to only serving the UI / API / WS roles.